bamboo/integration_tests/common_code.py

```diff
@@ -0,0 +1,154 @@
+import csv, os, pprint, re, time
+
+# Set up the command ##########################################################
+
+def get_model_option(dir_name, model_folder, model_name):
+    return '--model=%s/model_zoo/models/%s/model_%s.prototext' % (dir_name, model_folder, model_name)
+
+def get_reader_option(dir_name, reader_name):
+    return '--reader=%s/model_zoo/data_readers/data_reader_%s.prototext' % (dir_name, reader_name)
+
+def get_optimizer_option(dir_name, optimizer_name):
+    return '--optimizer=%s/model_zoo/optimizers/opt_%s.prototext' % (dir_name, optimizer_name)
+
+def get_command(dir_name, model_folder, model_name, executable, output_file_name):
+    # N => number of nodes.
+    # p => partition.
+    # t => timeout period, in minutes.
+    # n => number of processes to run. MPI Rank.
+    # n / procs_per_model = how many models should be made. (It must be true that n >= procs_per_model).
+    # num_epochs => number of epochs.
+    # data_reader_percent => how much of the data to use.
+    model_option = get_model_option(dir_name, model_folder, model_name)
+    optimizer_option = get_optimizer_option(dir_name, 'adagrad')
+    if model_name == 'alexnet':
+        reader_option = get_reader_option(dir_name, 'imagenet')
+        command = 'salloc -N 16 -p pbatch -t 600 srun -n 32 %s %s %s %s --num_epochs=20 --data_reader_percent=0.10 > %s' % (executable, model_option, reader_option, optimizer_option, output_file_name)
+    elif model_name == 'conv_autoencoder_mnist':
+        reader_option = get_reader_option(dir_name, 'mnist')
+        command = 'salloc -N 1 -p pbatch -t 60 srun -n 2 %s %s %s %s --num_epochs=5 > %s' % (executable, model_option, reader_option, optimizer_option, output_file_name)
+    elif model_name == 'conv_autoencoder_imagenet':
+        reader_option = get_reader_option(dir_name, 'imagenet')
+        command = 'salloc -N 16 -p pbatch -t 600 srun -n 32 %s %s %s %s --num_epochs=20 --data_reader_percent=0.10 > %s' % (executable, model_option, reader_option, optimizer_option, output_file_name)
+    elif model_name == 'lenet_mnist':
+        reader_option = get_reader_option(dir_name, 'mnist')
+        command = 'salloc -N 1 -p pbatch -t 60 srun -n 2 %s %s %s %s --num_epochs=5 > %s' % (executable, model_option, reader_option, optimizer_option, output_file_name)
```

Comment:
Let's change this partition to debug per Nikoli's suggestion. I reckon we can lower the time limit as well since this run takes like a minute total.